File: Python/1018.py
notes = int(input())
hundred = notes // 100
fifty = (notes % 100) // 50
twenty = ((notes % 100) % 50) // 20
ten = (((notes % 100) % 50) % 20) // 10
five = ((((notes % 100) % 50) % 20) % 10) // 5
two = (((((notes % 100) % 50) % 20) % 10) % 5) // 2
one = (((((notes % 100) % 50) % 20) % 10) % 5) % 2

print(notes)
print(f"{hundred} nota(s) de R$ 100,00")
print(f"{fifty} nota(s) de R$ 50,00")
print(f"{twenty} nota(s) de R$ 20,00")
print(f"{ten} nota(s) de R$ 10,00")
print(f"{five} nota(s) de R$ 5,00")
print(f"{two} nota(s) de R$ 2,00")
print(f"{one} nota(s) de R$ 1,00")

File: JavaScript/1018.js
var input = require("fs").readFileSync("/dev/stdin", "utf8");
var lines = input.split("\n");

var notes = parseInt(lines.shift());

var hundred, fifty, twenty, ten, five, two, one;
console.log(notes);
hundred = Math.floor(notes / 100);
notes = notes % 100;
console.log(hundred, "nota(s) de R$ 100,00");
fifty = Math.floor(notes / 50);
notes = notes % 50;
console.log(fifty, "nota(s) de R$ 50,00");
twenty = Math.floor(notes / 20);
notes = notes % 20;
console.log(twenty, "nota(s) de R$ 20,00");
ten = Math.floor(notes / 10);
notes = notes % 10;
console.log(ten, "nota(s) de R$ 10,00");
five = Math.floor(notes / 5);
notes = notes % 5;
console.log(five, "nota(s) de R$ 5,00");
two = Math.floor(notes / 2);
notes = notes % 2;
console.log(two, "nota(s) de R$ 2,00");
one = notes;
console.log(one, "nota(s) de R$ 1,00");
